tests/tck/features/optimizer/CollapseProjectRule.feature

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Push Filter down LeftJoin rule
+
+  Background:
+    Given a graph with space named "nba"
+
+  Scenario: Collapse Project
+    When profiling query:
+      """
+      MATCH (v:player)
+      WHERE v.age > 38
+      WITH v, v.age AS age, v.age/10 AS iage, v.age%10 AS mage,v.name AS name
+      RETURN iage
+      """
+    Then the result should be, in any order:
+      | iage |
+      | 4    |
+      | 4    |
+      | 4    |
+      | 4    |
+      | 4    |
+      | 4    |
+      | 4    |
+      | 4    |
+      | 4    |
+      | 4    |
+      | 3    |
+    And the execution plan should be:
+      | id | name        | dependencies | operator info |
+      | 16 | Project     | 14           |               |
+      | 14 | Filter      | 7            |               |
+      | 7  | Project     | 6            |               |
+      | 6  | Project     | 5            |               |
```

Comment:
could these projects not be collapsed ?

Reply:
In order to avoid multiple calculations of the expression, this situation is disabled.